src/main/java/voyager/SnippetFactory.java

```diff
@@ -0,0 +1,563 @@
+        stopwords.add("this");
+        stopwords.add("that");
+        stopwords.add("these");
+        stopwords.add("those");
+        stopwords.add("am");
+        stopwords.add("is");
+        stopwords.add("are");
+        stopwords.add("was");
+        stopwords.add("were");
+        stopwords.add("be");
+        stopwords.add("been");
+        stopwords.add("being");
+        stopwords.add("have");
+        stopwords.add("has");
+        stopwords.add("had");
+        stopwords.add("having");
+        stopwords.add("do");
+        stopwords.add("does");
+        stopwords.add("did");
+        stopwords.add("doing");
+        stopwords.add("a");
+        stopwords.add("an");
+        stopwords.add("the");
+        stopwords.add("and");
+        stopwords.add("but");
+        stopwords.add("if");
+        stopwords.add("or");
+        stopwords.add("because");
+        stopwords.add("as");
+        stopwords.add("until");
+        stopwords.add("while");
+        stopwords.add("of");
+        stopwords.add("at");
+        stopwords.add("by");
+        stopwords.add("for");
+        stopwords.add("with");
+        stopwords.add("about");
+        stopwords.add("against");
+        stopwords.add("between");
+        stopwords.add("into");
+        stopwords.add("through");
+        stopwords.add("during");
+        stopwords.add("before");
+        stopwords.add("after");
+        stopwords.add("above");
+        stopwords.add("below");
+        stopwords.add("to");
+        stopwords.add("from");
+        stopwords.add("up");
+        stopwords.add("down");
+        stopwords.add("in");
+        stopwords.add("out");
+        stopwords.add("on");
+        stopwords.add("off");
+        stopwords.add("over");
+        stopwords.add("under");
+        stopwords.add("again");
+        stopwords.add("further");
+        stopwords.add("then");
+        stopwords.add("once");
+        stopwords.add("here");
+        stopwords.add("there");
+        stopwords.add("when");
+        stopwords.add("where");
+        stopwords.add("why");
+        stopwords.add("how");
+        stopwords.add("all");
+        stopwords.add("any");
+        stopwords.add("both");
+        stopwords.add("each");
+        stopwords.add("few");
+        stopwords.add("more");
+        stopwords.add("most");
+        stopwords.add("other");
+        stopwords.add("some");
+        stopwords.add("such");
+        stopwords.add("no");
+        stopwords.add("nor");
+        stopwords.add("not");
+        stopwords.add("only");
+        stopwords.add("own");
+        stopwords.add("same");
+        stopwords.add("so");
+        stopwords.add("than");
+        stopwords.add("too");
+        stopwords.add("very");
+        stopwords.add("s");
+        stopwords.add("t");
+        stopwords.add("can");
+        stopwords.add("will");
+        stopwords.add("just");
+        stopwords.add("don");
+        stopwords.add("should");
+        stopwords.add("now");

```

Comment:
Couldn't we just initialize this using an array + the hashset constructor? Seems a lot less complicated and probably has a lot less startup cost compared to this method.

Reply:
It'd be syntactically better, but would it really be cheaper? The HashSet ctor requires a Collection not an array, so we'd need to dump the array into a list using Arrays.asList(), and I don't know if that'd be cheaper than just adding to the HashSet like we do now.

Reply:
Changed my mind, even if performance is similar the readability is enough of a plus for this to happen. Commit c82da18 fixes this.